core/src/com/unciv/logic/map/TileMap.kt

```diff
@@ -94,25 +90,28 @@ class TileMap {
                     var currentY = centerY - distance
 
                     for (i in 0 until distance) { // From 6 to 8
-                        yield(getIfTileExistsOrNull(currentX, currentY))
+                        getIfTileExistsOrNull(currentX, currentY)?.also { yield(it) }
```

Comment:
I understand why you're using .also, but I don't think it's very obvious what you're doing... is there a different way of doing this, perhaps?